spec/linter-tidy-spec.js

```diff
@@ -4,8 +4,27 @@ import * as path from 'path';
 
 const lint = require(path.join('..', 'lib', 'main.coffee')).provideLinter().lint;
 
+// TODO: Implement variables and functions related to the main linter package.
```

Comment:
I'll drop the new tests. You're right - that's on the consumer. If something wasn't right on our end, it would be obvious.